<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_notas</class>
 <widget class="QWidget" name="form_notas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Courier 10 Pitch&quot;;
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(170, 255, 127);
color: rgb(0, 0, 0);
background-color: rgb(117, 0, 88);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Notas e informes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Título:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_notas_titulo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Fecha:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dte_fec_notas">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Detalle:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txt_detalle_notas">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Tipo de nota:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbx_nota_tipo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <item>
            <property name="text">
             <string>Informe de auditoría</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reunión C.A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Informe de sindicatura</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Actas de asambleas</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Otros</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="boton_notas_guardar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Listado de Notas</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tw_lista_notas">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <column>
          <property name="text">
           <string>Titulo</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Detalle</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo Nota</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Detalle:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txt_notas_busqueda">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="boton_notas_imprimir">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
         </property>
         <property name="text">
          <string>Imprimir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
